conda_env/cli/main.py

```diff
@@ -35,6 +35,14 @@
 from . import main_update
 
 
+# TODO: This belongs in a helper library somewhere
```

Comment:
That's not to say that you couldn't do it there, just if you want it to be handled automatically it should happen before the parse even happens.